nzedb/ReleaseExtra.php

```diff
@@ -210,6 +216,15 @@ public function addSubs($releaseID, $subsID, $subslanguage)
 		}
 	}
 
+	/**
+	 * @param $releaseID
+	 * @param $uniqueid
+	 */
+	public function addUID($releaseID, $uniqueid)
+	{
+		$this->pdo->queryExec(sprintf("INSERT IGNORE INTO release_unique (releases_id, uniqueid) VALUES (%s, UNHEX('%s'))", $releaseID, $uniqueid));
```

Comment:
We are moving away from sprintf() arguments wherever possible in favor of double quoting the query string and inserting the values directly.

In this case, drop the sprintf and replace your %s with {$releaseID} and {$unqiueid} respectively.  Then look at how much cleaner PHPStorm says the query is when you're done.

```php
$this->pdo->queryExec("
   INSERT IGNORE INTO release_unique (releases_id, uniqueid)
   VALUES ({$releaseID}, UNHEX('{$uniqueid}'))"
);
```